tests/src/Kernel/fixtures/markup_rendering_patterns/card.yml

```diff
@@ -31,6 +34,8 @@ card_horizontal_with_image:
       'h5.card-title': 'Title card'
       'h6.card-subtitle': 'Subtitle card'
       'div.card-text': 'Text card with longer text which is supported'
+      '.card-body > div > span.text-muted:nth-child(1)': 'DIGIT'
```

Comment:
Can we add `.card-body > div > span.text-muted: 2` in the count assertions, so we make sure only 2 are rendered.